torch/fx/proxy.py

```diff
@@ -206,9 +206,16 @@ def create_proxy(self, kind: str, target: Target, args: Tuple[Any, ...], kwargs:
         else:
             proxy = proxy_factory_fn(node)
 
+        def _find_user_frame2_4(stack_trace):
```

Comment:
this is a very simplified function. it is a patch I did in my tracer that inherits form torch.fx tracer, and makes it work in both 2.3 and 2.4 that is the 2_4. you should probably call the original find_user_frame in the new code.